README.md

```diff
@@ -19,12 +19,31 @@ Include the package locally in your repository.
 
 `npm install dotenv-webpack --save`
 
+### Description
+
+`dotenv-webpack` wraps `dotenv` and `Webpack.DefinePlugin`. As such, it overwrites existing anyexisting `DefinePlugin` configurations. Also, like `DefinePlugin`, it does a text replace in the resulting bundle for any instances of `process.env`. 
+
+As such, if used with `React` in production, make sure to include `NODE_ENV=production` inside your `.env` file.
+If used with Node, this is not compatible with any calls to `process.env`.
```

Comment:
Along with the comment above, I would say you can nix this comment as we are a webpack plugin and not exclusively a "node" plugin (though it runs in the node environment).